source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -711,16 +711,16 @@ namespace aspect
                   }
                 beta[indices.back()] = 0.0;
 
-                // Now compute the crystal rate of deformation tensor.
-                for (unsigned int i = 0; i < 3; ++i)
+                // Now compute the crystal rate of deformation tensor. equation 4 of Kaminski&Ribe 2001
+                // rotation_matrix_transposed = inverse of rotation matrix
+                // (see Engler et al., 2024 book: Intro to Texture analysis chp 2.3.2 the rotAtion MAtrix)
```

Comment:
This is a funny spelling of "rotAtion MAtrix". Is this how it's spelled in the book?

Reply:
Thanks for the comment, just submitted another commit update the typo to Rotation Matrix